lib/mongo/collection.rb

```diff
@@ -371,6 +373,7 @@ def insert_many(documents, options = {})
     #   should be executed in order.
     # @option options [ Hash ] :write_concern The write concern options.
     #   Can be :w => Integer, :fsync => Boolean, :j => Boolean.
+    # @option options [ true, false ] :bypass_document_validation Whether or not to skip document level validation.
```

Comment:
These documentation lines should be shorter. I would put them on two lines. Generally, 80 characters is the max line length in Ruby.